lib/storage.js

```diff
@@ -73,6 +75,7 @@ async function putRendition(rendition) {
         if (rendition.instructions.name) {
             const newPath = path.join(rendition.directory, rendition.instructions.name);
             await fs.copyFile(rendition.path, newPath);
+            rendition.path = newPath; // assign newly created path
```

Comment:
Is this needed? It don't think any code in sdk looks at `rendition.path` after this line (since it doesn't upload in the WORKER_TEST_MODE case here, the rendition handling is done with this copyFile()).

Reply:
Oh but it does.

Reply:
Where?

Reply:
Even if something reads it, why should it be updated here. This is like a different version of upload, just by doing copyFile instead of HTTP PUT. Anything the sdk would need from the rendition it can get from the existing rendition path (source of the copy).

Reply:
Some validation. You'll get a rendition_failed event if not updated (allthough the rendition was created).

Reply:
(^ This is empiric. It's what I observed while debugging a worker. Otherwise I wouldn't have done the change).

Reply:
Nevermind. It was my debug code that did some stuff (it was never in this PR). So I fixed it.